src/services/constructions/assign_to_owner.js

```diff
@@ -0,0 +1,14 @@
+/**
+ * Ensure that only owners of constructions
+ * have access to them.
+ */
+async function assignToOwner(context) {
+    const owner = context.params.user;
```

Comment:
I would strongly recommend using destructuring of function parameters to avoid such nested chaining. Also, this prevents from creating temporary references to the objects within the given parameter. Also, this gives nice, consistent information to the reader about the expected structure of a parameter. More information is available [here](https://github.com/airbnb/javascript#destructuring).

Reply:
But how would you change this particular example?

Reply:
In this case it would look something like this:

```javascript
async function assignToOwner({ params: { user: owner }, app: { service } }) { /* snip */ }
```